api/app/src/routes/settings.ts

```diff
@@ -78,4 +103,48 @@ router.post(
   })
 );
 
+/** ---------------------------------------------------------------------------
+ * GET /settings/webhook
+ *
+ * Get webhook status information.
+ *
+ * @return {WebhookStatusDTO} The Webhook status information or empty body if no
+ * no settings defined.
+ */
+router.get(
+  "/webhook",
+  asyncHandler(async (req: Request, res: Response) => {
+    const id = getCxIdOrFail(req);
+    const settings = await getSettings({ id });
+    if (!settings) return res.status(status.OK).send({});
+    const { processing, failure } = await countFailedAndProcessingRequests(id);
+    res
+      .status(status.OK)
+      .json(
+        WebhookStatusDTO.fromEntity(
+          settings.webhookEnabled,
+          settings.webhookStatusDetail,
+          processing,
+          failure
+        )
+      );
+  })
+);
+
+/** ---------------------------------------------------------------------------
+ * POST /settings/webhook/retry
+ *
+ * Retries failed webhook requests.
+ *
+ * @return {200} indicating retry being processed.
+ */
```

Comment:
Let's add this to our docs?

Reply:
Added to the downstream PR to keep this simpler: https://github.com/metriport/metriport/pull/14/commits/e136a461c32b87d4050ae9a67af8186b38275325